commons/src/main/java/com/powsybl/commons/config/VersionConfig.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.config;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class VersionConfig {
+
```

Comment:
i don't think it is a good name. Maybe ConfigVersion or VersionNumber? (Note: Version is already used as Interface in powsybl)